modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -3032,6 +3035,12 @@ public void testChangeSnapshotTransferRateInRuntime() throws Exception {
         // Set transfer rate to unlimited.
         assertEquals(EXIT_CODE_OK, (int)propFunc.apply(0));
 
+        // Add release time of BasicRateLimiter#acquire() for the given rate.
+        BasicRateLimiter limiter = new BasicRateLimiter(rate);
+
+        limiter.acquire(SNAPSHOT_LIMITED_TRANSFER_BLOCK_SIZE_BYTES);
```

Comment:
Why did you extract `rate` into a separate variable but don't use it?

Reply:
Hi. `acquire` is about block size, not about rate.